extensions/funqy/funqy-server-common/runtime/src/main/java/io/quarkus/funqy/runtime/FunctionInvoker.java

```diff
@@ -76,15 +74,11 @@ public boolean hasInput() {
         return inputType != null;
     }
 
-    public Class getInputType() {
+    public Type getInputType() {
         return inputType;
     }
 
-    public Type getInputGenericType() {
-        return inputGenericType;
-    }
-
-    public Class getOutputType() {
+    public Type getOutputType() {
```

Comment:
@patriot1burke  I think that having both: `inputType` and `inputGenericType` is redundant. You can get `inputType` from `inputGenericType` by `getRawType()`.